### PR TITLE
Add support for Telegram join/part

### LINKED ISSUE #206: Create Join/Part Handlers
TeleIRC needs to be able to monitor when users join and leave a channel or group, and send that message over to the other platform. This specific issue 

- [x] Create handlers to monitor joining/parting messages on Telegram

### LINKED ISSUE #270: Write unit tests for join/part/quit handlers
# Summary

Unit tests do not currently exist for joining, parting, and quitting an IRC channel.

# Background

**Is your improvement related to a problem? Please describe**:

After writing these handlers, it is unclear if we have handled all edge cases. We should have unit tests to thoroughly test this functionality.

**Describe the solution you'd like**:

Full set of unit tests to catch as many edge cases as we can.

**Describe alternatives you've considered**:

No alternatives.

# Details

The owner of this issue needs to think of what the joining, parting, and quitting handlers should support, and not support. Then, build unit tests around the desired and undesired functionality to make sure error handling exists in the appropriate locations.

# Outcome

Unit testing encompasses the join, part, and quit handlers. Allows us to feel safe knowing these handlers are thoroughly tested.

### PR DESCRIPTION
**Pull request by Tjzabel**
## What this PR Does
* Allows support for IRC users to see when users join the Telegram channel
* If users do not wish to see this functionality, these messages can be toggled off

## What this looks like
```
Tim (@Tjzabel) has [joined|left] the Telegram Group!
```
or
```
Tim has [joined|left] the Telegram Group!
```
Closes #206

### CODE REVIEW

**Comment by Tjzabel** [comment]
@jwflory do you know what the `codeclimate/diff-coverage` is?

File: internal/handlers/telegram/handler_test.go [hunk 1]
```diff
@@ -0,0 +1,132 @@
+package telegram
```
Comment (kennedy) [inline internal/handlers/telegram/handler_test.go:1]: How would you do this with package telegram_test??
